adapters/socketio_berkeley.c

```diff
@@ -50,6 +50,7 @@
 
 #define SOCKET_SUCCESS                 0
 #define INVALID_SOCKET                 -1
+#define SOCKET_SEND_FAILURE            -1
```

Comment:
>SOCKET_SEND_FAILURE [](start = 8, length = 19)

SOCKET_SEND_ERROR?

Reply:
Yes, that's not to confuse about errno, which is another channel for error codes.